src/components/Cards/MeetingCard/MeetingCard.tsx

```diff
@@ -1,87 +1,53 @@
 import React from "react";
 import styled from "@emotion/styled";
-import Card from "../Base/Base";
-import colors from "../../../styles/colors";
-import { fontSizes } from "../../../styles/fonts";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+const DateOverlay = styled.p`
+  position: absolute;
+  z-index: 1;
+  color: white;
+  font-size: 30px;
+  font-weight: bolder;
+  top: 75%;
+  left: 2%;
+`;
```

Comment:
There is a part of me that thinks maybe we shouldn't have the date overlay on the image and just have it as content on the card... I mean we are getting an accessibility warning for a reason...

Reply:
Yeah, the warning just means it couldn't figure out if there's enough color contrast, but manually checking there is enough color contrast. I like getting rid of the overlay too. The only advantage I could think of for keeping the overlay is creating more room for the other content.

Reply:
Agree but I think we can make the cards larger if we need more space AND we can fix the accessibility warning. Which imo, the accessibility warning is important to address.

Reply:
Another option would be to simply add a background to the overlay, thus saving room and guaranteeing accessibility.

Reply:
This is a very quick change so I'll push something up so you guys can give feedback on the design.